### PR TITLE
The final batch of en_US improvements (hopefully) 

### PR DESCRIPTION
**Pull request by dominichayesferen**
### Check list
- [ ] Referenced all related issues
- [ ] Have tested the modifications

---

Just a few more after looking through the latest update with an emphasis on finding inconsistencies or sentences that don't sound quite right as a native speaker, and an added change to work around a bug that prevents the browser name placeholder from translating to 'Floorp' in question dialogs.

### CODE REVIEW

**Comment by surapunoyousei** [comment]
b

File: floorp/browser/locales/en-US/floorp.ftl
Comment (regularenthropy) [inline floorp/browser/locales/en-US/floorp.ftl:609]: Why { -brand-short-name } got removed?

File: floorp/browser/locales/en-US/floorp.ftl
Comment (regularentropy) [inline floorp/browser/locales/en-US/floorp.ftl]: "Use the red download icon when downloading" will be better

File: floorp/browser/locales/en-US/floorp.ftl
Comment (regularentropy) [inline floorp/browser/locales/en-US/floorp.ftl]:
```suggestion
  .tooltiptext = Keep using current width on this Panel
```

Comment (regularentropy) [reply floorp/browser/locales/en-US/floorp.ftl]: In my opinion, the use of "this" here is too abstract.

File: floorp/browser/locales/en-US/floorp.ftl
Comment (regularentropy) [inline floorp/browser/locales/en-US/floorp.ftl:37]:
```suggestion
tab-sleep-timeout-minutes-value = Tabs will be put into sleep after being inactive for (minutes)
```

File: floorp/browser/locales/en-US/floorp.ftl
Comment (regularentropy) [inline floorp/browser/locales/en-US/floorp.ftl]:
```suggestion
 .label = Enable the SimpleUI Download Manager
```
Better to leave it as it was, because "SimpleUI Download Manager" is more descriptive and specific than the new label.

**Comment by regularenthropy** [comment]
Everything else seems fine.

**Review by regularenthropy — APPROVED** [review]

**Comment by regularenthropy** [comment]
Everything else seems fine for me.

File: floorp/browser/locales/en-US/floorp.ftl
Comment (dominichayesferen) [reply floorp/browser/locales/en-US/floorp.ftl]: to be honest, I had no idea it was called 'SimpleUI' as the string before my commits similarly called it 'simple downloads management interface'.

File: floorp/browser/locales/en-US/floorp.ftl
Comment (dominichayesferen) [reply floorp/browser/locales/en-US/floorp.ftl]: 'the current width' sounds better in a grammatical context, so I changed it accordingly.

File: floorp/browser/locales/en-US/floorp.ftl
Comment (dominichayesferen) [reply floorp/browser/locales/en-US/floorp.ftl:37]: 'Tabs will sleep' sounds more natural than 'Tabs will be put into sleep', given the connotation used of tabs suspending/sleeping, plus the shorter string means it's less likely to need to wrap into a second line.

I changed it to what I changed it to here in this merge request as I realised after the prior PR's merging that "put to sleep" sounds too much like an action with a seriously morbid meaning to it.

**Comment by surapunoyousei** [comment]
Thanks. I'm not a native English speaker. because of this, Your enhancement patch is helpful for me.

Could I merge it?

**Comment by dominichayesferen** [comment]
> Thanks. I'm not a native English speaker. because of this, Your enhancement patch is helpful for me.
> 
> Could I merge it?

I would wait for @regularenthropy's response to the final unresolved convo, to be honest, just so there's a universal agreement about the remaining string.

**Comment by dominichayesferen** [comment]
@regularenthropy Should I assume your thumbs up is a sign I should hit Resolve Conversation?

**Comment by regularenthropy** [comment]
> > Thanks. I'm not a native English speaker. because of this, Your enhancement patch is helpful for me.
> > Could I merge it?
> 
> I would wait for @regularenthropy's response to the final unresolved convo, to be honest, just so there's a common agreement about the remaining string.

The last string was based on a localization from MS Edge. But, if it has "morbid meaning", than ok, let's leave it as it is.

**Comment by dominichayesferen** [comment]
Alright, you can merge now if you want to

**Comment by dominichayesferen** [comment]
> > > Thanks. I'm not a native English speaker. because of this, Your enhancement patch is helpful for me.
> > > Could I merge it?
> > 
> > 
> > I would wait for @regularenthropy's response to the final unresolved convo, to be honest, just so there's a common agreement about the remaining string.
> 
> The last string was based on a localization from MS Edge. But, if it has "morbid meaning", than ok, let's leave it as it is.

Ah, I see what you mean now, and to explain the difference: 'go to sleep', Edge's string, is an action regarding, or a command to do, the act of falling asleep (depending on the context).
However, 'put to sleep'... that, and I wish I remembered it did before I put it in its original pull request, is slang for euthanasia.

**Pull request merged**
